# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/tests/mochitest/mochitest.toml
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/credentialmanagement/identity/tests/mochitest/mochitest.toml
--- a/dom/credentialmanagement/identity/tests/mochitest/mochitest.toml	Thu Jun 05 00:15:19 2025 +0300
+++ b/dom/credentialmanagement/identity/tests/mochitest/mochitest.toml	Thu Jun 05 00:19:41 2025 +0300
@@ -1,6 +1,7 @@
 [DEFAULT]
 prefs = [
   "dom.security.credentialmanagement.identity.enabled=true",
+  "dom.security.credentialmanagement.identity.heavyweight.enabled=true",
   "dom.security.credentialmanagement.identity.select_first_in_ui_lists=true",
   "dom.security.credentialmanagement.identity.reject_delay.enabled=false",
   "privacy.antitracking.enableWebcompat=false", # disables opener heuristic